tests/unit/tracking.tcl

```diff
@@ -172,6 +174,7 @@ start_server {tags {"tracking network"}} {
     }
 
     test {Invalidations of previous keys can be redirected after switching to RESP3} {
+        r HELLO 2
```

Comment:
is this a bugfix for the test?
it meant to swtich protocol, but because other tests were added before it, it didn't get to test what it meant to?

Reply:
Before this test, r is RESP3. For this test, I think it should switch from 2 to 3.